ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/ConnectIntegrationTest.java

```diff
@@ -239,6 +252,68 @@ public void shouldReturnError() {
         containsString("Connector mock-connector already exists"));
   }
 
+  @Test
+  public void shouldReadAndWriteTimestampsToConnect() throws UnsupportedEncodingException {
+    // Given:
+    create("mock-source", ImmutableMap.<String, String> builder()
+        .put("connector.class", "org.apache.kafka.connect.tools.VerifiableSourceConnector")
+        .put("topic", "foo")
+        .put("throughput", "5")
+        .put("id", "123")
+        .put("topic.creation.default.replication.factor", "1")
+        .put("topic.creation.default.partitions", "1")
+        .build());
+    assertThatEventually(
+        () -> ((ConnectorList) ksqlRestClient.makeKsqlRequest("SHOW CONNECTORS;").getResponse()
+            .get(0)).getConnectors().get(0).getState(),
+        is("RUNNING (1/1 tasks RUNNING)"));
```

Comment:
This is the `given` section. I don't think we should have any checks here. But you can move this inside the `create` method after the request is called. Other tests do this often, so it's good if you want to make sure that `create` will mock the right thing.

Btw, how does the assert know the running connector is the one previously created?